client/landing/jetpack-cloud/components/with-server-credentials-form/test/index.js

```diff
@@ -191,28 +191,30 @@ describe( 'useWithServerCredentials HOC', () => {
 		expect( advancedSection.innerHTML ).toContain( 'Hidden content!' );
 	} );
 
-	it( 'should use rewindState to prefill the form', async () => {
```

Comment:
Non-blocking: We tend to use `.skip` if we need to temporary disable a test.

```
// Temporary disabled to prevent build failures in master. Needs to be re-enabled and properly fixed (more info: p1589962543224200-slack-calypso).
it.skip( 'should use rewindState to prefill the form', async () => {
```

Reply:
Thanks for the tip